apps/dashboard/src/components/editor/toolbar.js

```diff
@@ -74,46 +65,37 @@ const Toolbar = ( { projectId } ) => {
 
 	return (
 		<ToolbarSlot className="block-editor__crowdsignal-toolbar">
-			<Button
-				className="is-crowdsignal"
+			<ToolbarButton
```

Comment:
I think I missed something here... so, creating a style and then using it with `as={ Component }` is enough? We don't need to "wrap" the component with a styled component? Sounds good, it's just that I never seen it like this before

Reply:
Yeah. It's quite useful - we actually use it like that a lot in our block-editor package where we want `RichText` etc. be styled as well.

There's also a `cx` helper in `@emotion` which kind of acts like `classnames` for these. But in general, such cases would be more about the styled component being defined as a certain element, and us wanting to render it as a different one or as a React component with just the same styles applied to it.